services/motion/motion.go

```diff
@@ -1,4 +1,4 @@
-// Package motion implements an motion service.
+// Package motion is the service that allows you to plan movements with obstacle avoidance.
```

Comment:
lets just say `plan and execute movements`.  Obstacle avoidance is only one of the features we support